### PR TITLE
Create entry on pytorch tensors sum

### LINKED ISSUE #5061: [Term Entry] PyTorch Tensors .sum()
### Reason/inspiration (optional)

Add a new term entry about the `.sum()` method for Tensors. Place your file under the file path `docs/content/pytorch/concepts/tensors/terms/sum/sum.md` 

It should include the following sections:

- A "Syntax" section with a \```pseudo code block
- An "Example" section with a \```py code block
- A "Codebyte Example" section with a \```codebyte/python code block

Please refer to the [concept entry templates](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by arisdelacruz**
### Description

This is an attempt to create an entry on pytorch's sum function (on tensors)

### Issue Solved

Closes #5061

### Type of Change

- Adding a new entry

### Checklist

- [ / ] All writings are my own.
- [ / ] My entry follows the Codecademy Docs style guide.
- [ / ] My changes generate no new warnings.
- [ / ] I have performed a self-review of my own writing and code.
- [ / ] I have checked my entry and corrected any misspellings.
- [ / ] I have made corresponding changes to the documentation if needed.
- [ / ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ / ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ / ] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Comment by arisdelacruz** [comment]
just created a pull request, thanks

File: content/pytorch/concepts/tensors/terms/sum/sum.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  - 'Data Science'
+Tags:
+  - 'AI'
+  - 'Deep Learning'
+  - 'Functions'
+  - 'Machine Learning'
+CatalogContent:
+  - 'intro-to-py-torch-and-neural-networks'
+  - 'py-torch-for-classification'
+---
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensors/terms/sum/sum.md:15]:
```suggestion
---
Title: '.sum()'
Description: 'Computes the sum of all elements in a tensor.'
Subjects:
  - 'AI'
  - 'Data Science'
Tags:
  - 'AI'
  - 'Deep Learning'
  - 'Functions'
  - 'Machine Learning'
CatalogContent:
  - 'intro-to-py-torch-and-neural-networks'
  - 'py-torch-for-classification'
---
```

Comment (mamtawardhani) [reply content/pytorch/concepts/tensors/terms/sum/sum.md:15]: Keep the description small and to the point. Additional stuff is added in the introductory paragraph below

File: content/pytorch/concepts/tensors/terms/sum/sum.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+```pseudo
+torch.sum(input, *, dim=None, keepdim=False, out=None)
+```
+
+The `.sum()` function takes one required parameter `input`, which represents the input tensor whose elements are to be summed. It also has four other optional parameters that can be used to customize the output tensor if needed:
+
+- `dim`: Specifies the dimension along which the sum is to be computed. If `dim` is not specified, the sum is computed over all elements of the input tensor.
+- `keepdim`: If set to `True`, the output tensor will have the same number of dimensions as the input tensor.
+- `out`: Stores the output into existing tensor (if there is any).
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensors/terms/sum/sum.md:29]: Can we add both the syntax here? Like this:

To calculate the sum of all elements, the following syntax is used:

```pseudo
torch.sum(input, *, dtype=None)
```

- `input`: The input tensor.
- `dtype` (optional): The data type of the output tensor.

To calculate the sum along specific dimensions, the following syntax is used:

```pseudo
torch.sum(input, *, dim=None, keepdim=False, out=None)
```

- `input`: The input tensor.
- `dim` (optional): Specifies the dimension(s) along which the sum is computed. If not specified, the sum is computed over all elements.
- `keepdim` (optional): If set to `True`, the output tensor retains dimensions of size one for the reduced dimensions. The default is `False`.
- `out` (optional): A tensor to store the output into. If not provided, a new tensor is created.

File: content/pytorch/concepts/tensors/terms/sum/sum.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+## Example
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensors/terms/sum/sum.md:31]:
```suggestion
## Example 1
```

File: content/pytorch/concepts/tensors/terms/sum/sum.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+Here is the output for the above example:
+
+```shell
+tensor(10)
+```
+
+In this example, the sum of all elements in the input tensor `[[1, 2], [3, 4]]` is computed, resulting in a tensor with a single element `10`. The sum is calculated as `1 + 2 + 3 + 4 = 10`.
+
+The `.sum()` function can also be used to compute the sum along a specific dimension of the tensor. For example, to compute the sum along the rows of a 2D tensor, you can specify the `dim` parameter as `0` or `1` depending on the desired dimension.
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensors/terms/sum/sum.md:53]:
```suggestion
```

Comment (mamtawardhani) [reply content/pytorch/concepts/tensors/terms/sum/sum.md:53]: Since we will be adding in the syntax, let us avoid it from here

File: content/pytorch/concepts/tensors/terms/sum/sum.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensors/terms/sum/sum.md:54]:
```suggestion
## Example 2

```

File: content/pytorch/concepts/tensors/terms/sum/sum.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+print(sum_tensor)
+```
+
+Here is the output for the above example:
+
+```shell
+tensor(10)
+```
+
+In this example, the sum of all elements in the input tensor `[[1, 2], [3, 4]]` is computed, resulting in a tensor with a single element `10`. The sum is calculated as `1 + 2 + 3 + 4 = 10`.
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensors/terms/sum/sum.md:51]: Let us add the descriptions before the code

File: content/pytorch/concepts/tensors/terms/sum/sum.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+print(sum_rows)
+```
+
+Here is the output for the above example:
+
+```shell
+tensor([4, 6])
+```
+
+In this example, the sum is computed along the rows of the input tensor `[[1, 2], [3, 4]]`, resulting in a tensor with two elements `[4, 6]`. The sum along the rows is calculated as `[1 + 3, 2 + 4] = [4, 6]`.
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensors/terms/sum/sum.md:73]: Let us add the descriptions before the code

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @arisdelacruz thank you for contributing to Codecademy Docs! TThe entry is nicely written 😄 

I've suggested a few modifications, could you please review and modify them at your earliest convenience? Feel free to discuss any concerns, thank you! 😄

**Comment by arisdelacruz** [comment]
@mamtawardhani 

i hope i was able to come to terms with the suggested edits

thanks much

**Review by mamtawardhani — APPROVED** [review]
Thank you for contributing @arisdelacruz 😄 

The entry looks good for a second round of review! 🚀

**Review by Sriparno08 — APPROVED** [review]
Looks good, @arisdelacruz!
